server/src/Application/Dtos/Incoming/DtoControllers/CourseDtoController.cs

```diff
@@ -0,0 +1,18 @@
+using StudyZen.Domain.Entities;
```

Comment:
don't see any value in having these "controller" classes; we could just pass a `Course` to `CourseDto` constructor or have a static method in `CourseDto` which given a `Course` returns a `CourseDto`